src/routes/home/components/announcementDialog/data/announcementData.js

```diff
@@ -1,21 +1,15 @@
 import { Card } from '../../../../../../tests/e2e/fixtures/cards';
 
 export const announcementData = {
-  id: 'recyclerFivesPermanent',
-  activatorText: 'Recycler Fives are here to stay!',
-  title: 'Recycler Fives',
+  id: 'opponentHandRevealed',
+  activatorText: 'announcement.activatorText',
+  title: 'announcement.title',
   imgSrc: null,
-  displayCards: [Card.FIVE_OF_CLUBS, Card.FIVE_OF_DIAMONDS, Card.FIVE_OF_HEARTS, Card.FIVE_OF_SPADES],
+  displayCards: [Card.FIVE_OF_CLUBS, Card.ACE_OF_DIAMONDS, Card.KING_OF_HEARTS, Card.EIGHT_OF_SPADES],
```

Comment:
Let's replace the `displayCards` with an imgSrc to showcase the actual game board